bin/snare

```diff
@@ -147,15 +152,20 @@ if __name__ == '__main__':
     parser.add_argument("--update-timeout", help="update snare every timeout ", default='24H')
     parser.add_argument("--server-header", help="set server-header", default='nginx/1.3.8')
     parser.add_argument("--no-dorks", help="disable the use of dorks", type=str_to_bool,  default=True)
-    parser.add_argument("--log-dir", help="path to directory of the log file", default='/opt/snare/')
+    parser.add_argument("--path", help="path to save the page to be cloned", required=False, default='/opt/')
+    parser.add_argument("--page-path", help="Use only if the path not same as that of snare. Path to cloned pages",
```

Comment:
maybe instead of adding new CLI argument (we have so many now and it's easy to be confused) it's better to allow use an absolute path for page-dir? What do you think? Smth like if page-dir is relative, merge it with base_path, otherwise use absolute

Reply:
Yeah, that seems only logical.